src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -34,77 +75,85 @@
   [rn/view {:style (style/loading-message theme)}])
 
 (defn- loading-image
-  [{:keys [theme gradient-color-index]}]
-  [gradients/view
-   {:theme           theme
-    :container-style (style/loading-image theme)
-    :color-index     gradient-color-index}])
+  [{:keys [theme gradient-color-index loader-opacity]}]
+  [reanimated/view
+   {:style (style/loading-image-with-opacity theme loader-opacity)}
+   [gradients/view
+    {:theme           theme
+     :container-style (style/loading-image theme)
+     :color-index     gradient-color-index}]])
 
 (defn- card-details
   [{:keys [community? avatar-image-src collectible-name theme state set-state]}]
-  [rn/view {:style style/card-details-container}
-   (cond (not (:avatar-loaded? state))
-         [rn/view {:style {:flex-direction :row}}
-          [loading-square theme]
-          [loading-message theme]]
-
-         community?
-         [:<>
-          [preview-list/view
-           {:type :communities
-            :size :size-20}
-           [avatar-image-src]]
-          [rn/view {:style {:width 8}}]
-          [text/text
-           {:size   :paragraph-1
-            :weight :semi-bold
-            :style  style/card-detail-text}
-           collectible-name]])
-
-   [rn/view
-    {:style (style/avatar-container (:avatar-loaded? state))}
-    [:<>
-     [collection-avatar/view
-      {:size        :size-20
-       :on-load-end #(set-state (fn [prev-state] (assoc prev-state :avatar-loaded? true)))
-       :image       avatar-image-src}]
-     [rn/view {:style {:width 8}}]]
-    [text/text
-     {:size            :paragraph-1
-      :weight          :semi-bold
-      :ellipsize-mode  :tail
-      :number-of-lines 1
-      :style           style/card-detail-text}
-     collectible-name]]])
+  (let [loader-opacity            (reanimated/use-shared-value 1)
```

Comment:
@OmarBasem - good spot on this section, was all a bit wrong but I have since refactored and fixed it. (I think it was my issue to begin with 😅 )

Reply:
No worries, thanks for fixing :)